ui/src/pages/user_profile_page/UserProfilePage.tsx

```diff
@@ -0,0 +1,108 @@
+import { createSignal, JSXElement, Show } from 'solid-js'
+import { useUser } from '../../context'
+import { clsx } from 'clsx'
+
+import api from '../../api'
+import { ChangePasswordForm } from './ChangePasswordForm'
+import { Modal } from '../../components/Modal'
+
+export function UserProfilePage(): JSXElement {
+  const { user } = useUser()
+  const [openPasswordModal, setOpenPasswordModal] = createSignal(false)
+
+  return (
+    <>
+      <div class="mt-4 ml-10">
+        <Show when={user().isAdmin}>
+          <p class="text-lg text-success mr-4 mb-6 col-span-4">
+            <i class="fa-solid fa-screwdriver-wrench mr-2" />
+            This user is an admin
+          </p>
+        </Show>
+
+        <div>
+          <table class="table">
```

Comment:
The table is rendered on top of the dropdown menu at the top right corner now. The dropdown needs a higher z order maybe? (or lower?)